Add a "how long to wait until the next webcam" progress pie chart

ultrafez opened the pull request:
Add an indicator showing whether the webcam is currently cycling between cameras, or paused on one camera.

ultrafez commented:
@hexwab, would you be interested in doing a quick code review on this PR? It's non-trivial so I'd rather have a second pair of eyes look over it rather than me just merge it in straight away.

iMartyn commented:
@ultrafez I think all that makes sense, although I'd be tempted to split it into more than one commit.

One to change from Timeout to Interval
One to add the functions from the connector to the controller
One to add the new html and css
and a final one to add the js that implements the pie chart

ultrafez commented:
I've split the commits up and added a bit more info to each commit description. It would be tricky to split the switch from intervals to timeouts from the change that moves the functions, so I've left them as one commit. Hopefully it's clear enough what's going on now!
